src/odemis/driver/npmc.py

```diff
@@ -358,7 +364,9 @@ def MotorOff(self, axis):
 
     def GetMotionDone(self, axis_n):
         # Return true or false based on if the axis is still moving.
-        return bool(self._sendQuery("%d MD?" % axis_n))
+        done = int(self._sendQuery("%d MD?" % axis_n))
```

Comment:
Do you still need this extra logging? Normally, the value reported by `MD?` should already be logged via `_sendQuery()`. If so, just revert back to a single simple line.